src/hotspot/share/opto/opaquenode.cpp

```diff
@@ -107,27 +107,59 @@ const Type* OpaqueNotNullNode::Value(PhaseGVN* phase) const {
 }
 
 Node* OpaqueTemplateAssertionPredicateNode::Identity(PhaseGVN* phase) {
-  if (phase->C->post_loop_opts_phase()) {
-    // Template Assertion Predicates only serve as templates to create Initialized Assertion Predicates when splitting
-    // a loop during loop opts. They are not used anymore once loop opts are over and can then be removed. They feed
-    // into the bool input of an If node and can thus be replaced by true to let the Template Assertion Predicate be
-    // folded away (the success path is always the true path by design).
-    return phase->intcon(1);
```

Comment:
Moved to `Value()`. There is also no guarantee that this is an existing node (though `ConI #1` should probably always exist at that point).